src/payments/services/payment.service.ts

```
@@ -219,13 +223,96 @@ export class PaymentService {
           paymentIntentId: intentInTransaction.id,
           transactionId: transaction.id,
           status: webhookEvent.status,
+          userId: intentInTransaction.userId,
         };
       },
     );
 
+    // Generate certificates after successful payment (outside transaction to avoid blocking)
+    if (result.processed && result.status === 'success' && result.userId) {
```

ESLint

[error] Replace `'success'` with `"success"`

(prettier/prettier)